locale/po/grasswxpy_cs.po

```diff
@@ -20821,3 +20821,6 @@ msgstr "&Stáhnout"
 
 msgid "Fetch list of available modules from GRASS Addons SVN repository"
 msgstr "Stáhnout seznam dostupných modulů z GRASS Addons SVN repozitáře"
+
+#~ msgid "Animation already axists"
```

Comment:
Should be “exists”, no?

Reply:
In theory yes, but in practice it is dead old gettext message code as being `#~ msgid`

Reply:
Ok, I assumed it maybe was an out-of-sync artefact.

Reply:
Thanks for checking!